TSPL.docc/LanguageGuide/ErrorHandling.md

```diff
@@ -699,6 +699,204 @@ let photo = try! loadImage(atPath: "./Resources/John Appleseed.jpg")
   ```
 -->
 
+## Specifying the Error Type
+
+All of the examples above use the most common kind of error handling,
+where the errors that your code throws
+can be values of any type that conforms to the `Error` protocol.
+This approach matches the reality that
+you don't know ahead of time every error that could happen
+while the code is running,
+especially when propagating errors thrown somewhere else.
+This approach also reflects the fact that errors can change over time.
```

Comment:
"This approach" starts both this sentence and the prior one. Perhaps use this instead?
```suggestion
It also reflects the fact that errors can change over time.
```